docs/Makefile

```diff
@@ -22,7 +22,7 @@ merge-versions:
 	mkdir -p build/html
 	cp -R "build/1.0/html" "build/html/1.0"
 	cp -R "build/2.0/html" "build/html/2.0"
-	cp -R "root/" "build/html"
+	cp -R root/* "build/html"
```

Comment:
This change was needed because the `root` directory was being copied over as a directory instead of its contents.